packages/calcite-components/conventions/Testing.md

```diff
@@ -4,6 +4,8 @@ Components should have an automated test for any incoming features or bug fixes.
 
 We encourage writing expressive test cases and code that indicates intent. Use comments sparingly when the aforementioned can't be fully achieved. Keep it clean!
 
+<!-- TODO: Get suggestions for what this should be replaced with -->
+
```

Comment:
Interestingly, this is somewhat still applicable since Lumina provides a matching API. Maybe we mention this here for clarity?